ansible/modules/hashivault/hashivault_approle_role.py

```diff
@@ -36,16 +36,16 @@
             - Number of times any particular SecretID can be used.
     secret_id_ttl:
         description:
-            - Duration after which any SecretID expires.
+            - Duration after which any SecretID expires, in seconds
     enable_local_secret_ids:
         description:
             - If set, the secret IDs generated using this role will be cluster local.
     token_ttl:
         description:
-            - Duration to set as the TTL for issued tokens and at renewal time.
+            - Duration to set as the TTL for issued tokens and at renewal time, in seconds.
     token_max_ttl:
         description:
-            - Duration after which the issued token can no longer be renewed.
+            - Duration after which the issued token can no longer be renewed, in seconds.
```

Comment:
There was some code I thought to convert times to an idempotent value. I'm not sure if that would help here. This is a change to the API which would trigger a new major release since it is breaking.